.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: go
+go:
+  - 1.7
+install: make install-ci
+env:
+  global:
+    - TEST_TIMEOUT_SCALE=20 # Set higher timeouts for Travis
+    - PACKAGE=github.com/$TRAVIS_REPO_SLUG
```

Comment:
I think you can avoid that when using the global modifier: https://docs.travis-ci.com/user/environment-variables/#Global-Variables